src/utils/DockerComposeToServiceOverride.ts

```diff
@@ -45,4 +46,18 @@ export default class DockerComposeToServiceOverride {
 
         return override
     }
+
+    private static parseCapAdd(compose: IDockerComposeService) {
+        const override = {} as any
+        if (!!compose.cap_add && Array.isArray(compose.cap_add)) {
+            const capabilityAdd = compose.cap_add.map(cap => `CAP_${cap}`)
```

Comment:
Let's make this conditional if `cap` already starts with `CAP_`, don't add another one. Something like this:

```
compose.cap_add
.map(cap => cap.toUpperCase())
.map(cap => cap.startsWith(`CAP`)?cap:`CAP_${cap}`)
```